PerfTools/AllocMonitor/plugins/ThreadTracker.cc

```diff
@@ -0,0 +1,31 @@
+// -*- C++ -*-
+//
+// Package:     PerfTools/AllocMonitor
+// Class  :     ThreadTracker
+//
+// Implementation:
+//     [Notes on implementation]
+//
+// Original Author:  Christopher Jones
+//         Created:  Mon, 11 Nov 2024 22:54:23 GMT
+//
+
+// system include files
+
+// user include files
+#include "ThreadTracker.h"
+
+//
+// constants, enums and typedefs
+//
+
+//
+// static data member definitions
+//
+
+namespace allocMon {
```

Comment:
The `AllocMonitorBase` and `AllocMonitorRegistry` are in `cms::perftools` namespace. I'm not sure if `ThreadTracker` would fit well in the same namespace, but I'd like to avoid proliferation of namespaces used in this package.

Reply:
How about `cms::perftools::allocMon`?

Reply:
> How about `cms::perftools::allocMon`?

I'd be fine with that